mutation/mutation_fragment_v2.hh

```diff
@@ -152,9 +152,9 @@ private:
             partition_start _partition_start;
             partition_end _partition_end;
         };
+        const kind _kind;
```

Comment:
Better put it in the front, less chance for `data` accesses to span multiple cache lines for small fragments.

Reply:
Note: the first patch is kind of pointless, because it will get 8-byte aligned either way.